project/Dockerfile

```diff
@@ -8,6 +8,7 @@ WORKDIR /usr/src/app
 RUN python -m pip install --upgrade pip
 COPY ./requirements.txt .
 RUN pip install -r requirements.txt
+RUN python -m spacy download en_core_web_sm
```

Comment:
add 'en_core' to the requirements file instead of downloading it during the docker image creation step.